docs/concepts/serialization/structures.md

```diff
@@ -2,7 +2,7 @@
 
 ## Account {#serialization-standard-account}
 
-An Account is a structure that represented a user on a legacy Casper network. Alongside the Condor protocol release, `Accounts` were replaced with `AddressableEntities` of the type `Account`. The account structure consists of the following fields:
+An Account is a structure that represented a user on a legacy Casper network. Alongside the Casper 2.0 protocol release, `Accounts` were replaced with `AddressableEntities` of the type `Account`. The account structure consists of the following fields:
```

Comment:
This is not accurate, since we have turned off `AddressableEntities` in 2.0

Reply:
Thanks, @mssteuer . I have requested @darthsiroftardis 's inputs in removing addressable entities reference across the Docs.